platform/platform-app/src/main/resources/features.xml

```diff
@@ -776,6 +776,12 @@
         <bundle>mvn:ddf.platform.country/platform-country-converter-local/${project.version}</bundle>
     </feature>
 
+    <feature name="platform-usng4j" install="manual" version="${project.version}"
```

Comment:
My comment was based on my thinking that the manual/auto refers to the feature's behavior when it is put in the `deploy` directory, not it's behavior when the app is started. However, that may be incorrect.